torch/distributions/logistic_normal.py

```diff
@@ -0,0 +1,46 @@
+import torch
+from torch.distributions import constraints
+from torch.distributions.normal import Normal
+from torch.distributions.transformed_distribution import TransformedDistribution
+from torch.distributions.transforms import ComposeTransform, ExpTransform, StickBreakingTransform
+from torch.distributions.utils import _sum_rightmost
+
+
+class LogisticNormal(TransformedDistribution):
+    r"""
+    Creates a logistic-normal distribution parameterized by
+    `mean` and `std` where::
+
+        X ~ LogisticNormal(loc, scale)
+        Y = log(X / (1 - X)) ~ Normal(loc, scale)
```

Comment:
This math confuses me; it suggests that `X` is univariate, but you've implemented the more general multivariate version (e.g Bernoulli vs Categorical). Is there a simple way to define your `LogisticNormal` distribution?